src/assembly/package.xml

```diff
@@ -0,0 +1,52 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2
+          http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <!-- Assembles a package that can run given dependency packages. This is used to
+         construct packages OS packages where underlying libraries, like rest-utils, can be shared
+         -->
+    <id>package</id>
+    <formats>
+        <format>dir</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.parent.basedir}</directory>
+            <outputDirectory>share/doc/${project.artifactId}/</outputDirectory>
+            <includes>
+                <include>version.txt</include>
+                <include>COPYRIGHT*</include>
+            </includes>
+        </fileSet>
+        <fileSet>
+            <directory>${project.parent.basedir}</directory>
+            <outputDirectory></outputDirectory>
+            <includes>
+                <include>bin/*</include>
+            </includes>
+        </fileSet>
+        <fileSet>
+            <directory>${project.parent.basedir}/config</directory>
+            <outputDirectory>etc/${project.artifactId}</outputDirectory>
```

Comment:
pretty sure this is going to break down in at least a few cases. are we trying to completely generalize or just cover most cases? for example, blueway has at least 2 different packaging setups.